### PR TITLE
Survey status

### LINKED ISSUE #113: Survey status
Survey page should show status.
What do we want our status options to be? - glossary has WIP, Published, Closed, Archived right now.
Should only Admins be able to edit status?

### PR DESCRIPTION
**Pull request by rmarow**
resolves #113

### CODE REVIEW

File: usaon_vta_survey/forms.py [hunk 1]
```diff
@@ -83,6 +83,7 @@ def conv_String(self, field_args, **extra):
     ResponseObservingSystem: model_form(
         ResponseObservingSystem,
         only=common_response_object_fields,
+        field_args={'status': {'get_label': 'id'}},
```
Comment (MattF-NSIDC) [inline usaon_vta_survey/forms.py:86]: Should status be on the `Survey` form, not the `ResponseObservingSystem` form?

File: usaon_vta_survey/models/tables.py [hunk 2]
```diff
@@ -133,11 +133,11 @@ class Survey(BaseModel):
         nullable=False,
     )
 
-    # Do we want this to be a dropdown?
-    status = Column(
-        String(128),
+    status_id = Column(
+        String,
+        ForeignKey('status.id'),
+        default="work in progress",
```
Comment (MattF-NSIDC) [inline usaon_vta_survey/models/tables.py:139]: If we eventually change the status values (e.g. from `work in progress` -> `wip`) we'd need to remember to change this default string. Using `STATUSES[0]` would solve this problem, but would also create an implicit dependency on the order of the `STATUSES` list being correct. We could put a comment next to the constant list saying that the first element is the default so people editing the list have the context we need. We may also want to type `STATUSES` as `Final[list[str]]`.

**Review by MattF-NSIDC — CHANGES_REQUESTED** [review]

File: usaon_vta_survey/models/tables.py (hunk 2 above)
Comment (rmarow) [reply usaon_vta_survey/models/tables.py:139]: Great idea.

File: usaon_vta_survey/forms.py (hunk 1 above)
Comment (rmarow) [reply usaon_vta_survey/forms.py:86]: @MattF-NSIDC  copy pasta mistake!

Comment (rmarow) [reply usaon_vta_survey/forms.py]: removed this.

**Pull request merged**